integration-tests/features/steps/oc.py

```diff
@@ -83,3 +83,9 @@ def oc_service_exist(context, service_name):
     metadata = check_and_get_attribute(context.oc_result, "metadata")
     name = check_and_get_attribute(metadata, "name")
     assert name == service_name, "Returned service has wrong name {name}".format(name=name)
+
+
+@when(u'I delete all pods for the service {service_name}')
+def oc_delete_selected_pods(context, service_name):
+    """Delete selected pods."""
+    oc_delete_pods(service=service_name)
```

Comment:
oc_delete_pods(service=service_name) didn't work for me, I changed it to ` oc_delete_pods(service_name)`, but still I get this as result.
https://gist.github.com/GeetikaBatra/fbe75eb078c7f2293165f4122c8c6ef0

Reply:
https://github.com/fabric8-analytics/fabric8-analytics-common/blob/master/integration-tests/features/src/utils.py#L62 . The oc_delete_pod function doesnot take service as a formal parameter. I think we need to introduce service=None here .